applications/rag/workloads.tfvars

```diff
@@ -30,16 +30,16 @@ cloudsql_instance_region = "us-central1"
 # Creates a google service account & k8s service account & configures workload identity with appropriate permissions.
 # Set to false & update the variable `ray_service_account` to use an existing IAM service account.
 create_ray_service_account      = true
-ray_service_account             = "ray-system-account"
+ray_service_account             = "ray-sa"
 enable_grafana_on_ray_dashboard = false
 # Creates a google service account & k8s service account & configures workload identity with appropriate permissions.
 # Set to false & update the variable `rag_service_account` to use an existing IAM service account.
 create_rag_service_account = true
-rag_service_account        = "rag-system-account"
+rag_service_account        = "rag-sa"
 
 # Creates a google service account & k8s service account & configures workload identity with appropriate permissions.
 # Set to false & update the variable `jupyter_service_account` to use an existing IAM service account.
-jupyter_service_account = "jupyter-system-account"
+jupyter_service_account = "jupyter"
```

Comment:
Why only jupyter has no sa?

Reply:
SA name has 6 character minimum, 30 character maximum.
we prefix with IM deployment name for marketplace, prefer to keep it small so it doesn't get truncated.